### PR TITLE
Support for aeson-2.0

### PR DESCRIPTION
**Pull request by vshabanov**

### CODE REVIEW

**Comment by kk-kien** [comment]
Hi @23Skidoo, can you please merge this simple PR?

Some other packages depend on this library. To upgrade Aeson to 2.0 in other packages, this package should be changed first.

Thanks!

**Comment by 414owen** [comment]
I'd also like to see this merged. @tibbe any chance?

File: ekg-json.cabal [hunk 1]
```diff
@@ -24,10 +24,10 @@ library
   exposed-modules:
     System.Metrics.Json
   build-depends:
-    aeson >=0.4 && < 1.6,
-    base >= 4.6 && < 4.16,
+    aeson >= 2.0 && < 2.1,
+    base >= 4.6 && < 4.17,
```
Comment (jonathanlking) [inline ekg-json.cabal]:
```suggestion
    aeson >= 2.0 && < 2.2,
    base >= 4.6 && < 4.18,
```

(Support for GHC 9.4 and stackage nightly that supports 9.4, which uses aeson-2.1.2.1)

Comment (vshabanov) [reply ekg-json.cabal]: Done

**Comment by nh2** [comment]
@tibbe @23Skidoo @ejconlon

Who currently has merge permissions for the `ekg` repos?

Would you like some volunteers?

It would be great to get `ekg` back in shape, as it's very useful for production Haskell projects.

**Comment by nh2** [comment]
Just to highlight that there are many people waiting for this, among which many companies:

image

**Pull request closed without merging**